components/fxa-client/ios/FxAClient/RustFxAFFI.h

```diff
@@ -57,6 +57,9 @@ FxARustBuffer fxa_get_access_token(FirefoxAccountHandle handle,
                                    const char *_Nonnull scope,
                                    FxAError *_Nonnull out);
 
+FxARustBuffer fxa_check_authorization_status(FirefoxAccountHandle handle,
+                                   FxAError *_Nonnull out);
```

Comment:
nit: please align indentation of `FxAError` with that of `FirefoxAccountHandle` above it (similar to how the other functions above are aligned).